### PR TITLE
Settings fragment

### PR DESCRIPTION
**Pull request by andreitudorp**
Added:
- The settings fragment and activity
- Added general account settings 
- Added the ability to change your password
- Added the ability to change your email
- Added the ability to change you profile picture

Changed:
- Moved the logout and delete account from the profile screen

Solved 
- #11

### CODE REVIEW

File: app/src/main/java/com/example/musicmap/util/ui/ChangeEmailDialogFragment.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                    if (exception != null) {
+                        String message = exception.getMessage();
+                        newEmailInput.setError(message);
+                    }
+                }
+            });
+        }
+    }
+
+    private boolean checkEmail(String email) {
```
Comment (TPGamesNL) [inline app/src/main/java/com/example/musicmap/util/ui/ChangeEmailDialogFragment.java:74]: These functions are also found in RegisterFragment, extract them to a util class and make methods (with EditText param) for them

File: app/src/main/res/values/arrays.xml
Comment (TPGamesNL) [inline app/src/main/res/values/arrays.xml:1]: Where are these used?

File: app/src/main/res/layout/activity_settings.xml
Comment (TPGamesNL) [inline app/src/main/res/layout/activity_settings.xml:1]: I had this before, but is there a reason I can't see this layout in the visual editor? It says there are errors, no errors show up when I click it:
image

I had this before, if it's just my IDE then nvm

**Review by TPGamesNL — CHANGES_REQUESTED** [review]
btw resolve merge conflicts

File: app/src/main/res/values/arrays.xml
Comment (andreitudorp) [reply app/src/main/res/values/arrays.xml:1]: Nowhere. That file was generated when I tried creating the settings page. The files is removed now.

**Pull request merged**